libs/fixed_point/src/ifp256.sw

```diff
@@ -0,0 +1,306 @@
+library ifp256;
+// A wrapper library around the  type for mathematical functions operating with signed 256-bit fixed point numbers.
+use std::math::{Exponent, Power, Root};
+use ::ufp128::UFP128;
+
+pub struct IFP256 {
+    underlying: UFP128,
+    non_negative: bool,
+}
+
+impl From<UFP128> for IFP256 {
+    /// Creates IFP256 from UFP128. Note that IFP256::from(1) is 1 / 2^128 and not 1.
+    fn from(value: UFP128) -> Self {
+        Self {
+            underlying: value,
+            non_negative: true,
+        }
+    }
+
+    fn into(self) -> UFP128 {
+        self.underlying
+    }
+}
+
+impl IFP256 {
+    /// The size of this type in bits.
+    pub fn bits() -> u32 {
+        136
+    }
+
+    /// The largest value that can be represented by this type.
+    pub fn max() -> Self {
+        Self::from(UFP128::max())
+    }
+
+    /// The smallest value that can be represented by this type.
+    pub fn min() -> Self {
+        Self {
+            underlying: UFP128::min(),
+            non_negative: false,
+        }
+    }
+
+    pub fn zero() -> Self {
+        Self::from(UFP128::zero())
+    }
+
+    fn sign_reverse(self) -> Self {
```

Comment:
I think the same comments from the last PR apply to this one

Reply:
Fixed